src/components/Navbar/index.scss

```diff
@@ -54,4 +54,13 @@ nav.#{$baseClass} {
     position: fixed;
     top: 0;
   }
+
```

Comment:
navbar should have a transparent border without the border prop, that way the height could stay consistent

Reply:
fixed